crates/tauri-bundler/src/bundle/windows/templates/installer.nsi

```diff
@@ -637,6 +637,15 @@ Section Install
     WriteRegStr SHCTX "${UNINSTKEY}" $MultiUser.InstallMode 1
   !endif
 
+  ; Remove old main binary if it doesn't match new main binary name
+  ReadRegStr $0 SHCTX "${UNINSTKEY}" "MainBinaryName"
```

Comment:
does this delete the binary on each install? since the v2 installer also uses the same key.. maybe we should use a different one for v2?

Reply:
no this key will be removed when running the uninstaller, so it is only valid when doing updates. It has to be the same key for every installer (whether v1 or v2) so it can detect main binary name changes on updates